trieste/bayesian_optimizer.py

```diff
@@ -603,3 +558,173 @@ def optimize(
 
         record = Record(datasets, models, acquisition_state)
         return OptimizationResult(Ok(record), history)
+
+    def _write_summary_init(
```

Comment:
It might make sense to refactor bits of these out if and when we try to reuse them elsewhere, though I expect there'll remain some bits that are specific to BayesianOptimizer. Either way, don't think it should be part of this PR.

Reply:
ok, fair enough